### PR TITLE
MA: update legislators

### PR DESCRIPTION
**Pull request by jessemortenson**
* lower district `6th Worcester` is legitimately vacant per https://malegislature.gov/Legislators/Members/House

### CODE REVIEW

**Comment by jessemortenson** [comment]
@NewAgeAirbender lint failed here, I *think* only because of the missing legislator issue (see note in desc above)? Any idea how to handle this one?

**Comment by NewAgeAirbender** [comment]
> @NewAgeAirbender lint failed here, I _think_ only because of the missing legislator issue (see note in desc above)? Any idea how to handle this one?

Add the vacancy to the list in `settings.py` with an end date for next year

File: data/ma/legislature/Peter-J-Durant-db47013a-7653-4b44-8e11-9dbe4b85fec2.yml
Comment (NewAgeAirbender) [inline data/ma/legislature/Peter-J-Durant-db47013a-7653-4b44-8e11-9dbe4b85fec2.yml:1]: Need to add the ocd-id as another identifier on the file we're keeping or the merge will fail.

**Review by NewAgeAirbender — CHANGES_REQUESTED** [review]
Just update settings & add the deleted other id but the rest looks good!

**Pull request merged**